sdk/identity/azure-identity/azure/identity/aio/_authn_client.py

```diff
@@ -42,7 +42,7 @@ def __init__(
             DistributedTracingPolicy(),
         ]
         if not transport:
-            transport = AsyncioRequestsTransport(**kwargs)
+            transport = AioHttpTransport(**kwargs)
```

Comment:
Be sure to document that as breaking change, since now identity requires explicit installation of aiohttp to use the default

Reply:
Good point, thank you.